rvgo/fast/vm.go

```diff
@@ -670,8 +670,16 @@ func (inst *InstrumentedState) riscvStep() (outErr error) {
 		case 0: // 000 = ADDIW
 			rdValue = mask32Signed64(add64(rs1Value, imm))
 		case 1: // 001 = SLLIW
+			// SLLIW where imm[5] != 0 is reserved
+			if and64(imm, toU64(0x20)) != 0 {
```

Comment:
Basic question but how is `imm[5]` equivalent to `0x20`?

Reply:
Doing `and64(imm, 0x20)` gives us the 6th bit of imm, which we want to check(`imm[5]`) for